### PR TITLE
feat: add health and helloworld endpoint

### PR DESCRIPTION
**Pull request by saveroz**
Postman Tests Passed

Screenshot from 2021-04-15 13-40-36
Screenshot from 2021-04-15 13-40-23

### CODE REVIEW

**Comment by saveroz** [comment]
Able to handle 404 request

File: app/main.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+type Health struct {
+	Status string `json:"status"`
+}
+
+type HelloWorld struct {
+	Message string `json:"message"`
+}
+
+func healthHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method == "GET" {
```
Comment (carneles) [inline app/main.go:17]: can use constant http.MethodGet instead of "GET"

File: app/main.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+		w.Header().Set("Content-type", "application/json")
+		res := Health{Status: "healthy"}
+		json.NewEncoder(w).Encode(res)
```
Comment (owenyuwono) [inline app/main.go:20]: always handle error

**Review by owenyuwono — APPROVED** [review]

File: app/main.go (hunk 2 above)
Comment (saveroz) [reply app/main.go]: updated please check again @owenyuwono

**Review by khasburrahman — APPROVED** [review]

**Pull request closed without merging**